clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -2354,6 +2354,55 @@ public void testDeleteRecords() throws Exception {
         }
     }
 
+    @Test
+    public void testDescribeTopicsByIds() {
+        try (AdminClientUnitTestEnv env = new AdminClientUnitTestEnv(mockCluster(4, 0),
+                AdminClientConfig.RETRIES_CONFIG, "2")) {
```

Comment:
Just use `try (AdminClientUnitTestEnv env = mockClientEnv()) {` maybe? Setting cluster to 4 nodes and 2 retries does't sound relevant to this test